src/SFA.DAS.EmployerUsers.Web/Views/Shared/_Layout.cshtml

```diff
@@ -103,33 +103,7 @@
             </div>
         </div>
     </header>
-    <div class="header-user">

```

Comment:
Is this meant to be removed as its not mentioned in the ticket from what I can see?